<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #D5F2F2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="text_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	background-color: #7EC6F2
}

QPushButton{
	color: black;
	text-align:left;
	height:50px;
	border: none;
	padding-left: 10px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}

QPushButton:checked{
	background-color: #ffffff;
	color: #7EC6F2;
	font-weight: bold;
}
QPushButton:hover{
	background-color: #ffffff;
	color: #7EC6F2;
	font-weight: bold;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/newPrefix/eye.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Eye Tester</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="homeButton2">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Home</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/home.png</normaloff>:/newPrefix/home.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="statisticsButton2">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Statistics</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/user (1).png</normaloff>:/newPrefix/user (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exercisesButton2">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Exercises</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/settings.png</normaloff>:/newPrefix/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>209</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="signoutButton2">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sign Out</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/logout.png</normaloff>:/newPrefix/logout.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="main_menu_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="header_widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="menuButton">
            <property name="styleSheet">
             <string notr="true">border: none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/newPrefix/list.png</normaloff>:/newPrefix/list.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>540</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: #7EC6F2;</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="setup_window">
          <widget class="QLabel" name="Webcam">
           <property name="geometry">
            <rect>
             <x>350</x>
             <y>84</y>
             <width>640</width>
             <height>480</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0.477, y1:0, x2:0.506, y2:1, stop:0 rgba(242, 173, 126, 255), stop:1 rgba(255, 255, 255, 255));</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>100</y>
             <width>71</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>INFO</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="distance_guidance_label">
           <property name="geometry">
            <rect>
             <x>510</x>
             <y>20</y>
             <width>271</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Distance:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>180</y>
             <width>281</width>
             <height>281</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>16</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>• Move away from the monitor with the given value above the camera

• Stand still in the green zone for 3 seconds to start the test

• Not standing still might cancel the test</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="home_window">
          <widget class="QPushButton" name="start_test_button">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>520</y>
             <width>241</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>14</pointsize>
             <weight>10</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:20px;
background-color: #F2AD7E;
font: 87 14pt &quot;Segoe UI Black&quot;;
color:black;
</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="dioptre_input">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>470</y>
             <width>191</width>
             <height>41</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="dioptre_error_field">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>430</y>
             <width>241</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color: red;
font: 75 14pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>280</x>
             <y>390</y>
             <width>431</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>16</pointsize>
             <weight>10</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 87 16pt &quot;Segoe UI Black&quot;;</string>
           </property>
           <property name="text">
            <string>Inserati valoarea dioptriei dumneavoastra</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="statistics_window"/>
         <widget class="QWidget" name="exercises_window">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>420</x>
             <y>290</y>
             <width>181</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>26</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Exercises</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="test_window">
          <widget class="QLabel" name="visibleLetter1">
           <property name="geometry">
            <rect>
             <x>540</x>
             <y>70</y>
             <width>221</width>
             <height>381</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0.477, y1:0, x2:0.506, y2:1, stop:0 rgba(242, 173, 126, 255), stop:1 rgba(255, 255, 255, 255));
font: 87 16pt &quot;Segoe UI Black&quot;;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="testBotButton">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>500</y>
             <width>241</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>14</pointsize>
             <weight>10</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:20px;
background-color: rgba(193, 22, 99, 200);
font: 87 14pt &quot;Segoe UI Black&quot;;
background-color: qlineargradient(spread:pad, x1:0.506, y1:0.261, x2:0.500051, y2:1, stop:0 rgba(193, 22, 99, 200), stop:0.619318 rgba(193, 22, 99, 255));
color:white</string>
           </property>
           <property name="text">
            <string>Test</string>
           </property>
          </widget>
          <widget class="QLabel" name="current_distance_label">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>530</y>
             <width>261</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI Black</family>
             <pointsize>18</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Current Distance</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="icon_widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	
	background-color:#7EC6F2
}

QPushButton{
	color: black;
	height:50px;
	border: none;
	border-radius: 10px;
}

QPushButton:checked{
	background-color: #ffffff;
	color: #7EC6F2;
	font-weight: bold;
}
QPushButton:hover{
	background-color: #ffffff;
	color: #7EC6F2;
	font-weight: bold;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="images.qrc">:/newPrefix/eye.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="homeButton1">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/home.png</normaloff>:/newPrefix/home.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="statisticsButton1">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/user (1).png</normaloff>:/newPrefix/user (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exercisesButton1">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/settings.png</normaloff>:/newPrefix/settings.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>209</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="signoutButton1">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/newPrefix/logout.png</normaloff>:/newPrefix/logout.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>menuButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>text_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exercisesButton1</sender>
   <signal>toggled(bool)</signal>
   <receiver>exercisesButton2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>icon_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>homeButton2</sender>
   <signal>toggled(bool)</signal>
   <receiver>homeButton1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>45</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>homeButton1</sender>
   <signal>toggled(bool)</signal>
   <receiver>homeButton2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>statisticsButton1</sender>
   <signal>toggled(bool)</signal>
   <receiver>statisticsButton2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>statisticsButton2</sender>
   <signal>toggled(bool)</signal>
   <receiver>statisticsButton1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exercisesButton2</sender>
   <signal>toggled(bool)</signal>
   <receiver>exercisesButton1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
